### DISCUSSION TITLE
Loki live tailing, add test to datasource settings page

### DISCUSSION THREAD

**Discussion by gabor** [grafana/grafana#45582] [Observability]
right now it's hard to know why live-tailing is not working, perhaps we could have some way to test it on the datasource settings page.

(idea originally appeared in https://github.com/grafana/grafana/issues/19555)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
